addon/components/influencer-network-modal.js

```diff
@@ -1,10 +1,36 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { GLOBAL_SUPPORT_LINK } from '@upfluence/ember-upf-utils/resources/helpdesk-links';
+import { inject as service } from '@ember/service';
 
 const INFLUENCER_NETWORK_MODAL_COOKIE = 'upf_disable_influencer_modal';
 
 export default Component.extend({
+  currentUser: service(),
+  user: null,
+
+  init() {
+    this._super();
+    this.currentUser.fetch().then((user) => {
+      this.set('user', user);
+    });
+  },
+
+  displayInfluencerNetworkModal: computed(
+    'hasDisabledInfluencerNetworkModal',
+    'hideInfluencerNetworkModal',
+    'user.companies.{firstObject.billing_format,length}',
+    function () {
+      return (
+        !this.hasDisabledInfluencerNetworkModal &&
+        !this.hideInfluencerNetworkModal &&
+        this.user &&
+        this.user.companies &&
+        this.user.companies.firstObject.billing_format != 'bracket'
```

Comment:
You didn't check if `campanies` is empty here. And please use type check `!==`

Reply:
`this.user.companie` when he is empty, null, undefined = false in the condition and ok for the second part